spec/outputs/sns_spec.rb

```diff
@@ -2,17 +2,110 @@
 require "logstash/devutils/rspec/spec_helper"
 require 'logstash/outputs/sns'
 require 'logstash/event'
+require "logstash/plugin_mixins/aws_config"
+
+require "aws-sdk" # TODO: Why is this not automatically brought in by the aws_config plugin?
```

Comment:
You sure you need that?  The `register` method include the `aws-sdk-resource`,
But I see your point, I am not a fan of lazy require since they are lazy per default, because plugins are only loaded when they are defined in a running configuration.